packages/nc-gui-v2/components/general/Language.vue

```diff
@@ -37,7 +39,36 @@ onMounted(() => {
 </script>
 
 <template>
-  <a-dropdown class="select-none color-transition" :trigger="['click']">
+  <template v-if="subMenu">
+    <a-menu class="scrollbar-thin-dull min-w-50 max-h-90vh overflow-auto !py-0 rounded">
+      <a-menu-item
+        v-for="lang of languages"
+        :key="lang"
+        :class="lang === locale ? '!bg-primary/10 text-primary' : ''"
+        class="group"
+        :value="lang"
+        @click="changeLanguage(lang)"
+      >
+        <div
+          :class="lang === locale ? '!font-semibold !text-primary' : ''"
+          class="nc-project-menu-item capitalize group-hover:text-pink-500"
+        >
+          {{ Language[lang] || lang }}
+        </div>
+      </a-menu-item>
+
+      <a-menu-item>
+        <a
```

Comment:
I think it'll be better to just split the language component into two parts, one just contains the menu the other icon + menu - that way we avoid code duplication it stays more readable :)

Reply:
Sounds good will update into multiple component